src/components/element/Button.astro

```diff
@@ -4,7 +4,7 @@ interface Props {
   label: string;
   style: "primary" | "secondary";
   size: "md" | "lg";
-  icon: any; //for a button with an icon
+  icon?: string; //for any button with an icon
```

Comment:
Import 

```
import type { ImageMetadata } from "astro";
```
and use `ImageMetadata` here

Reply:
<img width="496" alt="Screenshot 2023-12-30 at 19 24 14" src="https://github.com/0xJamin/techconfafrica/assets/36636824/e0b9db8b-60de-464d-8afe-5f06962e68b4">
Is this the expected usage? on line 9?

Reply:
Yes this works

Reply:
```
interface Props {
...
icon?: ImageMetadata;


}
```